cryptostore/data/arctic.py

```diff
@@ -17,18 +17,23 @@ def __init__(self, connection: str):
         self.con = StorageEngines.arctic.Arctic(connection)
 
     def aggregate(self, data):
-        self.data = data
+        if isinstance(data[0], dict):
+            # Case `data` is a list or tuple of dict.
+            self.data = pd.DataFrame(data)
+        else:
```

Comment:
ok, I've tested the code, looks fine, but I'm not clear on what these isinstances are for in arctic, elastic, influx and parquet. Is this for the kafka case, which was unmodified?

Reply:
@yohplala

Reply:
Hi @bmoscon ,
Yes, but not only.
2 formats for data are also possible in `redis.py` depending the data types:
- l*_book_flatten generates a tuple (keys, generator)
- for the other  data type, the result is a list of dict (I have not modified the original format)

In the case of the generator, some backends need to know keys before starting to depile the generator, which is why I am keeping it along the generator.

So I am then testing in which cases we are in the different backends.
- either it is a list of dict (kafka, and for redis: ticker, trades, funding, open interest)
- either it is a tuple (keys, generator) (redis: order book)

Bests